app/src/ui/autocompletion/user-autocompletion-provider.tsx

```diff
@@ -31,7 +31,7 @@ export class UserAutocompletionProvider implements IAutocompletionProvider<IUser
 
   public async getAutocompletionItems(text: string): Promise<ReadonlyArray<IUserHit>> {
     const users = await this.gitHubUserStore.getMentionableUsersMatching(this.repository, text)
-    return users.map(u => ({ username: u.login, name: u.name }))
+    return users.map(u => ({ username: u.login, name: u.name || u.login }))
```

Comment:
This fallback may not be visually correct. I can take a more in-depth look at the AutoCompleter code if that is indeed the case.